apis/Google.Cloud.EntityFrameworkCore.Spanner/Google.Cloud.EntityFrameworkCore.Spanner/Storage/Internal/SpannerSqlGenerationHelper.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Text;
+using Google.Api.Gax;
+using Microsoft.EntityFrameworkCore.Storage;
+
+namespace Google.Cloud.EntityFrameworkCore.Spanner.Storage.Internal
+{
+    /// <summary>
+    /// </summary>
+    public class SpannerSqlGenerationHelper : RelationalSqlGenerationHelper
+    {
+        /// <summary>
+        /// </summary>
+        /// <param name="dependencies"></param>
+        public SpannerSqlGenerationHelper(RelationalSqlGenerationHelperDependencies dependencies)
+            : base(dependencies)
+        {
+        }
+
+        /// <inheritdoc />
+        public override void GenerateParameterName(StringBuilder builder, string name)
```

Comment:
this will change as I add more tests that cover using parameters in queries.  It pretty much has no effect whatsoever right now other than giving me a good place to see where the parameters are being generated.